tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -134,6 +154,7 @@ def _merge_items(self, first_items, remaining_items) -> dict:
         return deserialized_items
 
     def save(self, key_value_pairs) -> None:
+        log.debug(f"Adding to save queue: {key_value_pairs}")
```

Comment:
would this be too noisy in production?

Reply:
Err probably. I added a few of these while testing because our current logging only has a generic `_consume_save_queue saved X items in Ys`, I don't think I really intended these to make it into the PR 😐. Removed.